src/styles/_variables.scss

```diff
@@ -10,4 +10,5 @@ $link-color: #FFF;
 $finsec-ptx-navy: #11203A;
 $finsec-ptx-blue: #537190;
 $finsec-ptx-baby: #AAC7E1;
+$finsec-ptx-mid: #234C6B;
```

Comment:
mid what? perhaps give name that would make instant sense to someone that never worked with your code before? That's usually a good mindset. Having set that, I'll update the one above to `baby-blue` instead. Makes more sense.